CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
-## unreleased
+## 0.15.0
+* Breaking change: Major internal refactoring of public/private,
+  type definitions, templates, and warnings.   (#1524, #1539)
+* Breaking change: Allow mixins that call their super-classes. (#1555)
+* Breaking change: Anonymous libraries are now laid out on disk
+  differently to avoid conflicts (#1526)
+* Breaking change: The meaning of --auto-include-dependencies has changed to
+  include all libraries in any package depended on by this package (determined
+  by the .packages file) (#1524)
+* Breaking change: The meaning of --include and --exclude has changed to
+  require import paths for anonymous libraries, and accept them for other
+  libraries. (#1524)
+* The Interceptor class from the SDK is now cloaked (#1524)
+* Type parameters for classes now appear next to them on the library page
+  (#1558)
+* GFM-style tables are now supported in Dartdoc markdown (#1557, #1453)
```

Comment:
Looking though this changelog, this is a huge release. GFM tables, more correctness, new flags, changed meanings of existing flags. I'm wondering if we should do anything to make people better aware of the new features of the release? A blog / tweet / announcement?

@kevmoo @wmleler for thoughts